packages/renderer/src/lib/kube/KubeDetailsSummary.svelte

```diff
@@ -0,0 +1,144 @@
+<script lang="ts">
+import type { V1Pod } from '@kubernetes/client-node';
+import KubeDetailsSummaryVolumes from './KubeDetailsSummaryVolumes.svelte';
+export let pod: V1Pod | undefined;
+
+// https://github.com/kubernetes-client/javascript/issues/487
+if (pod?.metadata?.creationTimestamp) {
+  pod.metadata.creationTimestamp = new Date(pod.metadata.creationTimestamp);
+}
+
+if (pod?.status?.startTime) {
+  pod.status.startTime = new Date(pod.status.startTime);
+}
+</script>
+
+<div class="flex px-5 py-4 flex-col items-start hover:overflow-y-auto">
+  {#if pod}
+    <table class="w-full">
+      <tbody>
+        <tr>
+          <td class="py-2 px-4 text-lg pl-1 font-semibold text-purple-400" colspan="2">Pod {pod.metadata?.name}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Name</td>
+          <td class="py-2 px-4">{pod.metadata?.name}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Namespace</td>
+          <td class="py-2 px-4">{pod.metadata?.namespace}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Created</td>
+          <td class="py-2 px-4">{pod.metadata?.creationTimestamp}</td>
+        </tr>
+        <tr>
+          <td class="py-2 px-4">Labels</td>
```

Comment:
If there are no labels, then we see an empty Labels section, may be we can skip it completely